mscore/musescore.cpp

```diff
@@ -6685,6 +6685,11 @@ bool MuseScore::canSaveMp3()
 
 bool MuseScore::saveMp3(Score* score, const QString& name)
       {
+#ifndef USE_LAME
```

Comment:
This code is not needed here. This method redirects call to the `MuseScore::saveMp3(Score* score, QIODevice* device, bool& wasCanceled)`.